### ISSUE TITLE
You can remove the inline style from inputs 

### ISSUE THREAD

**Issue by ppetkow** [snamoah/react-datetime-range-picker#18]
https://github.com/snamoah/react-datetime-range-picker/blob/c821928cce1ce46abc05ca439756756d41454426/lib/index.jsx#L34

There is a function that passes inline style to datepicker input. Its hard to override style of this element. :(

**Comment by WizardCM** [comment]
Agreed. I vote simply change the order of the inputProps extend: that way the style only gets overwritten if explicitly specified. Having hardcoded styles that aren't in the provided stylesheet is painful as it forces developers to use `!important`.

**Comment by luizabentivoglio** [comment]
How can i remove the border from the input? Thank you

> Agreed. I vote simply change the order of the inputProps extend: that way the style only gets overwritten if explicitly specified. Having hardcoded styles that aren't in the provided stylesheet is painful as it forces developers to use `!important`.

How can i remove the border from the inputs?

**Comment by ppetkow** [comment]
If it comes from Props function you can only with !important in CSS file

**Comment by snamoah** [comment]
Great point guys... I'd appreciate if anyone could create a pull request with this fix. I can create a pr fix for this a bit later no one opts to do it
